perun-core/src/test/java/cz/metacentrum/perun/core/entry/AttributesManagerEntryIntegrationTest.java

```diff
@@ -8744,10 +8845,4945 @@ public void setAttributeRights() throws Exception {
 
 
 
+	// ============= 13. private methods for attribute dependencies logic ====================================
 
 
+//----------------------- VO attributes -------------------------//
 
-// PRIVATE METHODS ----------------------------------------------
+	@SuppressWarnings({"unchecked"})
```

Comment:
These suppressions are there because of the unchecked cast warning that is caused by converting the returned object of call testedMethod.invoke() to List<RichAttribute>.